src/Microsoft.Azure.WebJobs.Host/Blobs/BindableBlobPath.cs

```diff
@@ -10,7 +11,24 @@ internal static class BindableBlobPath
     {
         public static IBindableBlobPath Create(string pattern, bool isContainerBinding = false)
```

Comment:
>pattern [](start = 54, length = 7)

could pattern include { } and be these? 
{data.url} -- a url 
{container}  - a container 
{container}/{blobname}